.github/workflows/build.yaml

```diff
@@ -15,23 +15,86 @@ permissions: read-all
 
 env:
   AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
why do we need two of these? can we just use AUTH_TOKEN?

Reply:
GH_TOKEN works automatically with the `gh api` cli stuff

I didn't want to touch the AUTH_TOKEN since I didn't really know where it is used.

Reply:
it used to be used in bots.sh, but that usage is removed in this PR, so we can delete the AUTH_TOKEN one then